### DISCUSSION TITLE
Scope disconnects when trying to probe (6022BE)

### DISCUSSION THREAD

**Discussion by ghost** [OpenHantek/OpenHantek6022#166] [Q&A] [answered]
Hi all, 

I'm new to all this and am just wondering what could cause the scope to disconnect when trying to probe a DIY induction heater. 

I can read the gate input waveforms just fine when the half bridge in unpowered, however when I try to apply 12V for tuning/testing of the output coil openhantek freezes, the device disconnects. Then i need to restart openhantek. 

Do I need a special usb hub for isolation ? Ive already checked that the voltages I'm reading do not exceed the ratings on the scope, 

I checked with an analog scope that although I'm pushing 200khz from the driver, my coil rings at about 3Mhz. I am guessing it is the ringing that causes interference when i apply power to the drain / source of my half bridge.

**Comment by Ho-Ro** [comment, marked as answer]
> DIY induction heater
I'm pushing 200khz from the driver, my coil rings at about 3Mhz

Oh - oh, high current and high frequency -> if your ground connection isn't rock solid it calls all kind of trouble. 
You can use snap-on ferrites on all lines, position all cables carefully and avoid ground loops.

**Reply by ghost** [reply]
Thanks for the response, I will do further testing to see if I can get it to play nice with the high frequency and magnetic interference.
Ferrite snap ons  might be tricky to get hold of where I am but I believe I can get something figured out.

I'll post back on anything I figure out, cheers!
